aerospike/datadog_checks/aerospike/aerospike.py

```diff
@@ -198,8 +201,11 @@ def get_datacenters(self):
         return datacenters
 
     def get_client(self):
+        client_config = {'hosts': [self._host]}
+        if self._tls_config:
+            client_config['tls'] = self._tls_config
```

Comment:
(nit) Does that have the same effect ?

```suggestion
        client_config['tls'] = self._tls_config
```

Reply:
No, In one case the key exists with a None value and in the other one the key does not exist

Reply:
I meant from `aerospike.client` perspective. I guess it might be better to not set it, than set it to None anyway.